cmk/gui/plugins/wato/check_parameters/job.py

```diff
@@ -50,6 +50,17 @@ def _parameter_valuespec_job():
                  Age(title=_("Critical at"), default_value=0)
              ],
          )),
+        ("exit_codes",
+         ListOf(
```

Comment:
Add some help=_("...") to describe the exact behaviour

a) in case a single exit code is given two times. Which of the entries will result in the final monitoring state?
b) What is the default behaviour in case another exit code occurs than we have configured here?
c) What is the behaviour in case the I do not configure this attribute (add the sentence from the man page)